lib/airport.rb

```
@@ -1,2 +1,42 @@
+require './lib/plane.rb'
+
 class Airport
+
+attr_reader :capacity, :planes
+
+DEFAULT_CAPACITY = 10
+
+  def initialize
+    @planes = []
+    @capacity = DEFAULT_CAPACITY
+  end
+
+  def release_plane
+    return false if empty? || stormy?
```

This currently fails silently - I would raise an error in this situation.